scm/pr.go

```diff
@@ -64,6 +64,17 @@ type (
 		Color string
 	}
 
+	// Milestone the milestone
+	Milestone struct {
+		Number      int
+		ID          int
+		Title       string
+		Description string
+		Link        string
+		State       string
+		DueDate     *time.Time
```

Comment:
use `time.Time` instead of `*time.Time` and instead of `== nil` or `!= nil` we can use time.IsZero to check if set or not

Reply:
done